research/blueprint_algo/average_strategy_short_deck.py

```diff
@@ -74,7 +80,7 @@ def average_strategy(all_file_paths: List[str]) -> Dict[str, Dict[str, float]]:
 def cli(results_dir_path: str, write_dir_path: str):
     """Compute the strategy and write to file."""
     # Find all files to load.
-    all_file_paths = glob.glob(os.path.join(results_dir_path, "agent*.gz"))
+    all_file_paths = glob.glob(os.path.join(results_dir_path, "*_*"))
```

Comment:
what files are being missed by the agent*.gz glob?

Reply:
I used regular blueprint_short_deck algo. The previous might have been including the final dump, which is simply offline_strategy.gz. It might not be necessary for your multiprocess script, I don't know.

Reply:
I re-included strategy.gz below, which I was avoiding with the above